src/portal/src/i18n/lang/zh-cn-lang.json

```diff
@@ -813,32 +816,31 @@
     "SERVER_ERROR": "服务器出现内部错误，请求无法完成。",
     "INCONRRECT_OLD_PWD": "旧密码不正确。",
     "UNKNOWN": "未知",
-    "STATUS":"状态",
-    "START_TIME": "创建时间",
-    "UPDATE_TIME": "更新时间",
-    "LOGS":"日志",
-    "PENDING":"未开始",
-    "FINISHED":"已完成",
-    "STOPPED":"已停止",
-    "RUNNING":"执行中",
-    "ERROR":"错误",
+    "STATUS": "状态",
+    "START_TIME": "开始时间",
```

Comment:
why need to modify this update_time to endtime. It should not be modified in this pr.